mod/metrics/set/abstract/metric_answer/validate.rb

```diff
@@ -3,32 +3,39 @@
 end
 
 event :validate_value_type, :validate, on: :save, when: :standard? do
```

Comment:
I'm realizing that codeclimate isn't being very critical of complex blocks in event/view definitions.  I suspect that helps explain why it has generally seemed less critical of wikirate than decko...